### PR TITLE
Day1 - awesome books with es6

### PR DESCRIPTION
**Pull request by Nuga0909**
- Refactor methods and functions to arrow functions.
- Add a modules directory
- Use ES6 syntax
- Update html and css files

### CODE REVIEW

File: Modules/bookclass.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-restricted-globals */
```
Comment (mckent05) [inline Modules/bookclass.js]: - Please remove all disabled linters, remember that linters help to ensure that we write code that is clean and easy to maintain 😄 
HINT: The first disabled linter wants you to use the `this` keyword in your class methods, if you do not require the `this` keyword in your class method, use the `static` keyword before naming your class e.g ``` static storeBook () { ... } ```

File: index.js [hunk 2]
```diff
@@ -1,91 +1,55 @@
...
+      </div>
+      <hr class="divider" />`)
+  .join('');
+
+const removeBtn = document.querySelectorAll('.remove-btn');
+removeBtn.forEach((item, index) => {
+  item.addEventListener('click', () => {
+    const takeOut = new Book.Book(bookList[index].title, bookList[index].author);
+    takeOut.removeBook(takeOut.title);
+  });
```
Comment (mckent05) [inline index.js:46]: - Well done on the work done so far, but currently the UI is not getting updated when I add a new book, Please fix this 🙏 🙏. Please fix this
- Also please remember to save the added book in the local storage 👍 👍 

https://user-images.githubusercontent.com/73607512/205734823-86bdb275-6684-48e7-ac6c-a35c34731dc1.mp4

File: index.html [hunk 3]
```diff
@@ -1,15 +1,73 @@
...
+        <ul class="nav-list">
+          <li class="nav-list-item book-page"><a href="#display-Books">List <span class="divider">|</span></li></a>
+          <li class="nav-list-item add-page"><a href="# add-New">Add new <span class="divider">|</span></li></a>
+          <li class="nav-list-item info-page"><a href="#contact">Contact <span class="divider"></span></li></a>
+
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <p class="date"></p>
```
Comment (mckent05) [inline index.html:28]: - According to the project requirements, the current date is displayed below the navigation bar, currently your date is not displayed, please fix this 🙏 
Your project:
eye

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Nuga0909,

# STATUS: CHANGES REQUIRED ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there


## Highlights
- All linter checks are passing ✔️ 
- Your readme file is professional 👍 
- You used the right workflow for this project 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 2 above)
Comment (Nuga0909) [reply index.js:46]: Kindly load clone the repo to get it to work properly, as it does work here.

File: index.html (hunk 3 above)
Comment (Nuga0909) [reply index.html:28]: Kindly load clone the repo to get it to work properly, as it does work here.

File: index.js (hunk 2 above)
Comment (Owusu-Desmond) [inline index.js:46]: - [ ] As requested by the previous code reviewer currently, the UI is not getting updated when I add a new book, and also please remember to save the added book in the local storage. Please kindly fix this :pray:

File: index.html (hunk 3 above)
Comment (Owusu-Desmond) [inline index.html:28]: - [ ] As requested by the previous code reviewer, From the project requirements, the current date is displayed below the navigation bar, currently, your date is not displayed, please fix this :pray: 
image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Nuga0909 :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @Nuga0909 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f6669636f666f72756d732e6d796669636f2e636f6d2f74352f696d6167652f736572766572706167652f696d6167652d69642f343932363069463839303835363833444139334135422f696d6167652d73697

## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. 🙏
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  Good PR title/description 💯
 - [ ]  All linters passed 🥇
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**